Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/UpdateOpensearch.ts

```diff
@@ -37,6 +37,7 @@ export async function afterDeleteDocumentById(request: DeleteRequest, result: De
     await client.delete({ ...opensearchRequest, refresh: true });
   } catch (err) {
     Logger.error(`UpdateOpensearch.afterDeleteDocumentById`, request.traceId, err);
+    throw err;
```

Comment:
Only logging bothered me because opensearch will probably be out of sync as a result. Not sure of a better solution - 502 Bad Gateway would be a good one in a response, but this is async from the request.